plugins/janus_videoroom.c

```diff
@@ -2964,6 +2975,7 @@ static void *janus_videoroom_handler(void *data) {
 				publisher->video_active = FALSE;
 				publisher->data_active = FALSE;
 				publisher->recording_active = FALSE;
+				publisher->recording_pause = FALSE;
```

Comment:
This should be reset in `hangup_media` as well. As a side note, it might be useful to have this property returned when doing `query_session` as well, as we have a section related to recording in there when querying publishers.